pkg/workflows/default_workflow_service.go

```diff
@@ -36,22 +36,6 @@ func (s *DefaultWorkflowService) GetProfiles(b *browsers.Browser, query string)
 }
 
 func (s *DefaultWorkflowService) GetBookmarks(browser *browsers.Browser, query string) (bookmarks.BookmarkSlice, error) {
-	profileSlice, err := s.ProfileService.GetProfiles(browser)
-
-	if err != nil {
-		return nil, err
-	}
-
-	profileName := profiles.DefaultProfileName
-
-	if browser.ProfileName == "" {
-		if profile, err := profileSlice.FirstActive(); err == nil && profile.Name != "" {
-			profileName = profile.Name
-		}
-	}
-
-	browser.ProfileName = profileName
-
```

Comment:
Could you please explain why you have remove that block?

Reply:
You just need to set profile via `-profiles` command.

Reply:
This is the current problem:
First I made sure that the default browser is selected:
<img width="1440" alt="Screenshot 2021-11-20 at 10 05 24" src="https://user-images.githubusercontent.com/52125859/142720856-d930778c-c2fa-46aa-a7a6-9aa7ad7be84e.png">
<img width="1440" alt="Screenshot 2021-11-20 at 10 07 10" src="https://user-images.githubusercontent.com/52125859/142720885-55abd2f6-be51-44a4-b46a-aa267721072e.png">

Then I searched for a bookmark and the results were from the Default browser:
<img width="1440" alt="Screenshot 2021-11-20 at 10 05 31" src="https://user-images.githubusercontent.com/52125859/142720948-472dc7c2-3547-47d4-a1da-0ea5cbe76b2d.png">

But when i switched to Profile 1 an made sure that it was activated
<img width="1440" alt="Screenshot 2021-11-20 at 10 11 29" src="https://user-images.githubusercontent.com/52125859/142720986-c3438d71-097c-4d79-b194-9b3912597571.png">
It also saved the change to the browser.yml file:
<img width="1440" alt="Screenshot 2021-11-20 at 10 06 56" src="https://user-images.githubusercontent.com/52125859/142721015-148ac6a2-8beb-4aa1-bed9-25a61f9c41c9.png">

But after that when I searched for a bookmark and the results were still from the Default browser
<img width="1440" alt="Screenshot 2021-11-20 at 10 05 55" src="https://user-images.githubusercontent.com/52125859/142721050-e81ae6ee-d48d-4c8d-b524-70e723d47dbe.png">

Reply:
And removing that code resolved that problem